### ISSUE TITLE
Odoo Mobile App can't log in to self hosted Enterprise

### ISSUE THREAD

**Issue by phillipstack** [odoo/odoo#29107]
Impacted versions: Odoo 11.0+e

Steps to reproduce: 

-Install Odoo App
-Configure with https://yourselfhost.com
-Login

Current behavior: 
Your connection is not private or cant connect to server

Expected behavior: Log in and get to work

Video/Screenshot link (optional):

**Comment by phillipstack** [comment]
Our self hosted server was working for quite some time using the mobile app without issue. The ssl certificate is valid and not expired. No changes have been made however I see there was an update to the Android App ( i presume ios as well ) on November 28th relating to login. I am wondering if this is related.

**Comment by phillipstack** [comment]
The enterprise licence is valid until February and the issue is affecting android and apple devices of various makes and models.

**Comment by Yenthe666** [comment]
@adr-odoo can you check? Do you know about any updates that caused issues?
@phillipstack you should make an OPW under odoo.com/help as you have an enterprise license too. Response times are way better there.

**Comment by phillipstack** [comment]
I am the server administrator. There have been no updates on the server. I can ask the staff if they installed an app however I find this incredibly unlikely. 

Also, I was trapping incoming tcp traffic on the server and when we try to login from the mobile app there is absolutely no traffic. It really smells like a bug.

**Comment by phillipstack** [comment]
Login from the browser is normal, no issues.

**Comment by phillipstack** [comment]
Thanks @Yenthe666 appreciate the clarification.  I will do that if I get no traction here however unless I think this is a configuration issue I do not want to burn my clients support package for something that is beyond there control ( being that there is an app update Nov 28th and magically the app stops working ). Keep in mind, we have done nothing else to my knowledge.

**Comment by Yenthe666** [comment]
@phillipstack,

As far as I'm aware creating support tickets for bugs does not go from any package. My team has logged over 400 tickets this way without any costs, it falls under the enterprise warranty.

**Comment by phillipstack** [comment]
@Yenthe666 so I take care of this from the Enterprise Portal?

**Comment by phillipstack** [comment]
If you could provide me a link I would apprecate it.

**Comment by Yenthe666** [comment]
from https://www.odoo.com/help and ideally after you're logged in on the Odoo S.A. portal as you'll can link the ticket to you and to an enterprise license.

**Comment by phillipstack** [comment]
@Yenthe666 perfect thanks. Should this ticket be closed? Or should we wait and see?

**Comment by Yenthe666** [comment]
It is okay to keep it open, as long as you just keep track of both then 😉

**Comment by adr-odoo** [comment]
Hello @phillipstack , could you give me your ticket number?

**Comment by phillipstack** [comment]
Hi @adr-odoo the ticket number is 1913507

**Comment by phillipstack** [comment]
The issue was that I had restricted access to /web/database in my nginx configuration. I must have had the endpoint accessible when the apps were configured initially and closed it afterwards. I usually keep /web/database closed to avoid people trying to do anything suspicious. 

My guess is that the update to the app forced to mobile app to request this endpoint again and it failed. I am not sure why I did not see the traffic on my tcpdump ( maybe it came in on 80 not 443 ). Maybe an error on my part. 

So to resolve this faster 
-I should have identified the traffic ( didnt )
-Should have noticed Nginx was restricting access ( didnt )
-Would have been nice to have had a more descriptive error message

Anyways, I guess this is not a bug and is mostly my problem. Not sure where I could recommend a more descriptive error message.
